db/Dockerfile

```diff
@@ -1,66 +1,52 @@
-# based on https://github.com/arangodb/arangodb-docker/blob/official/alpine/3.4.0/Dockerfile
+# Edited from https://github.com/arangodb/arangodb-docker/blob/official/alpine/3.6.4/Dockerfile
+FROM alpine:3.10
+MAINTAINER Frank Celler <info@arangodb.com>
 
-# CHANGES:
-
-# - add extra GPG keyserver to return ipv4 addresses
-# - remove the conversion of endpoints from 127.0.0.1 to 0.0.0.0
-
-# - set ARANGO_NO_AUTH to 1
-# - copy dumps
-
-FROM alpine:3.8
-
-ENV ARANGO_NO_AUTH 1
-
-ENV ARANGO_VERSION 3.4.0
-ENV ARANGO_URL https://download.arangodb.com/arangodb34/DEBIAN/amd64
+ENV ARANGO_VERSION 3.6.4
+ENV ARANGO_NO_AUTH true
+ENV ARANGO_URL https://download.arangodb.com/arangodb36/DEBIAN/amd64
 ENV ARANGO_PACKAGE arangodb3_${ARANGO_VERSION}-1_amd64.deb
 ENV ARANGO_PACKAGE_URL ${ARANGO_URL}/${ARANGO_PACKAGE}
 ENV ARANGO_SIGNATURE_URL ${ARANGO_PACKAGE_URL}.asc
 
-RUN apk add --no-cache gnupg pwgen nodejs npm binutils && \
-    npm install -g foxx-cli && \
-    rm -rf /root/.npm
-
-RUN GPG_KEYS=CD8CB0F1E0AD5B52E93F41E7EA93F5E56E751E9B && \
-   ( gpg --keyserver ipv4.pool.sks-keyservers.net --recv-keys "$GPG_KEYS" \
-  || gpg --keyserver hkps://hkps.pool.sks-keyservers.net --recv-keys "$GPG_KEYS" )
-
-RUN mkdir /docker-entrypoint-initdb.d
-
 # see
-#   https://docs.arangodb.com/latest/Manual/Administration/Configuration/Endpoint.html
-#   https://docs.arangodb.com/latest/Manual/Administration/Configuration/Logging.html
-
-RUN cd /tmp                                && \
+#   https://www.arangodb.com/docs/3.6/programs-arangod-server.html#managing-endpoints
+#   https://www.arangodb.com/docs/3.6/programs-arangod-log.html
+# added "npm config set unsafe-perm true && \" because of this error:
+#   https://stackoverflow.com/a/52196681
+RUN apk add --no-cache gnupg pwgen nodejs npm binutils numactl numactl-tools && \
+    npm config set unsafe-perm true && \
+    npm install -g foxx-cli && \
+    rm -rf /root/.npm && \
+    gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys CD8CB0F1E0AD5B52E93F41E7EA93F5E56E751E9B && \
+    mkdir /docker-entrypoint-initdb.d && \
+    cd /tmp                                && \
     wget ${ARANGO_SIGNATURE_URL}           && \
     wget ${ARANGO_PACKAGE_URL}             && \
     gpg --verify ${ARANGO_PACKAGE}.asc     && \
     ar x ${ARANGO_PACKAGE} data.tar.gz     && \
     tar -C / -x -z -f data.tar.gz          && \
     sed -ri \
-        -e 's!^(file\s*=).*!\1 -!' \
+        -e 's!127\.0\.0\.1!0.0.0.0!g' \
```

Comment:
Does this override any firewall and expose ports to the internet?

Reply:
You're right!
This is original Dockerfile from arango team for version 3.6.4
https://github.com/arangodb/arangodb-docker/blob/official/alpine/3.6.4/Dockerfile
This line is removed now.